pkg/config/cli.go

```diff
@@ -244,15 +244,6 @@ func GetCmdOpts() CLIOptions {
 		K0sVars.DefaultStorageType = "kine"
 	}
 
-	// When CfgFile is set, verify the file can be opened
-	if CfgFile != "" {
-		if fd, err := os.Open(CfgFile); err != nil {
-			logrus.WithError(err).Fatalf("Cannot access config file (%s)", CfgFile)
-		} else {
-			_ = fd.Close()
-		}
-	}
-
```

Comment:
Yes, it's not necessarily a good solution. I'm just concerned that there be dragons if we remove this, since it might open up other edge cases in other commands that implicitly rely on this early check somehow. Anyway, I just wanted to point out that this has implications for virtually all subcommands.